src/pages/index.tsx

```diff
@@ -91,9 +91,12 @@ const Home: NextPage<ProjectProps> = ({ project }) => {
             <Image
               alt=""
               src={HomeImage3}
-              objectFit="cover"
-              layout="fill"
               placeholder="blur"
+              fill
+              sizes="100vw"
```

Comment:
This size is unreasonably large for some displays. Consider using media conditions. This applies to other images too.